README.md

```diff
@@ -0,0 +1,3 @@
+# SJSU Robotics Control Systems 2021
+
+Add more content later.
```

Comment:
Not sure why this file is in here. You may need to pull the latest master branch, then rebase off of that.